sdk/python/tests/test_repo_config.py

```diff
@@ -99,7 +101,7 @@ def test_bad_type():
             path: 100500
         """
         ),
-        expect_error="__root__ -> online_store -> path\n  str type expected",
+        expect_error="path\n  str type expected",
```

Comment:
Does this mean our error messages don't show the full path to the nested field that is misconfigured?

Reply:
I *think* i fixed this, let me check

Reply:
Yeah the original test still works so undid this change, thanks for pointing out